### ISSUE TITLE
Problem with IPC channel on windows

### ISSUE THREAD

**Issue by StarpTech** [apparatus/fuge-runner#15]
Hi,
it seems that there are problems to spawn a process with a IPC channel.
The current release doesnt work on windows.

error
```
? fuge> [service2 - 1696]: child_process.js:61
  p.open(fd);
    ^

Error: EBADF: bad file descriptor, uv_pipe_open
    at Error (native)
    at Object.exports._forkChild (child_process.js:61:5)
    at Function.startup.processChannel (node.js:844:10)
    at startup (node.js:45:15)
    at node.js:968:3
[api - 18392]: child_process.js:61
  p.open(fd);
```

When I remove the parameter it works.
https://github.com/apparatus/fuge-runner/blob/master/lib/processRunner.js#L103

https://github.com/apparatus/fuge/issues/79

**Comment by mcdonnelldean** [comment]
@mcollina @lucamaraschi Any ideas why we need that option set? I can add a windows check around it or remove completely.

**Comment by mcdonnelldean** [comment]
Landing soon,

I've added a handle for Windows to fix this issues.

**Comment by mcdonnelldean** [comment]
Fixed via #18
